shell/platform/android/platform_view_android.h

```diff
@@ -120,6 +120,12 @@ class PlatformViewAndroid final : public PlatformView {
     return platform_message_handler_;
   }
 
+  void SetIsRenderingToImageView(bool value) {
```

Comment:
If it is possible (I think it is) to have more than one FlutterImageViews active concurrently this should probably keep track of an integer count and only update the context when it hits 0 or != 0

Reply:
I think I could get that to happen if I overlay something on the platform view. Will try that out and see if I can repro.

Reply:
I played aroud with this a bit and I wasn't able to get this to happen in a simple example with an overlay over a platform view. In that case it looks like the overlay view is added as a child of the FlutterView and so their lifecycle is tied.

Perhaps this could happen in an add2app scenario with multiple flutter views sharing the same engine? Extermely cursed to think about those having platform views in them - but I'll make it a count anyway since that should be perfectly safe (TM).

Reply:
Thanks!